diff --git a/CMakeLists.txt b/CMakeLists.txt
index 44ca6f42..188aa91d 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -225,4 +225,4 @@ if(BUILD_TESTING)
   endforeach()
 endif()
 
-ament_package()
+ament_package(CONFIG_EXTRAS as2_core-extras.cmake)
\ No newline at end of file
diff --git a/as2_core-extras.cmake b/as2_core-extras.cmake
new file mode 100644
index 00000000..7d1c4491
--- /dev/null
+++ b/as2_core-extras.cmake
@@ -0,0 +1 @@
+find_package(GeographicLib REQUIRED CONFIG)
